reposcan/redhatcve/cvemap_controller.py

```diff
@@ -97,5 +97,7 @@ def store(self):
                 self._download_xml()
                 cvemap = self._load_xml(self.lastmodified)
                 self.cvemap_store.store(cvemap)
+        except Exception:  # pylint: disable=broad-except
```

Comment:
probably not neccessary here because errors are catched inside self.cvemap_store.store already

Reply:
:thinking: ok